specification/schemas/Service.json

```diff
@@ -33,6 +33,16 @@
               "example": "spring-pap",
               "description": "Slug of the service"
             },
+            "information": {
+              "type": "array",
+              "example": [
+                {
+                  "type": "notice",
```

Comment:
Do you have an idea on how many different `types` there will be? Maybe we can make this an enum?

Reply:
We don't know the rest of the types, so we will just add them later